MobSF/utils.py

```diff
@@ -468,6 +465,9 @@ def gen_sha256_hash(msg):
 def isFileExists(file_path):
     if os.path.isfile(file_path):
         return True
+    # This fix situation where a user just typed "adb" or another executable inside settings.py
+    if shutil.which(file_path):

```

Comment:
works on windows?

Reply:
Didn't test on Windows & Linux yet
(I guess I will send some fixes after I'll do it)